astropy/stats/bayesian_blocks.py

```diff
@@ -105,13 +105,13 @@ def bayesian_blocks(t, x=None, sigma=None,
     Event data:
 
     >>> t = np.random.normal(size=100)
-    >>> edges = bayesian_blocks(t, fitness='events', p0=0.01)
+    >>> edges = bayesian_blocks(t, fitness='events', gamma=1)
 
     Event data with repeats:
 
     >>> t = np.random.normal(size=100)
     >>> t[80:] = t[:20]
-    >>> edges = bayesian_blocks(t, fitness='events', p0=0.01)
+    >>> edges = bayesian_blocks(t, fitness='events', gamma=1)
```

Comment:
@crawfordsm @larrybradley - I'm not sure what the best way is to avoid the warning about p0 - this works but I'm not sure if it's a sensible value of gamma.

Reply:
Is this necessary now that the `p0` warning was removed?

Reply:
Yes this is fixed now so I've reverted.